Only in unica/app: footer
diff -ru dspace/app/header/header.component.html unica/app/header/header.component.html
--- dspace/app/header/header.component.html	2023-12-02 20:20:21.000000000 -0500
+++ unica/app/header/header.component.html	2023-11-11 10:23:47.000000000 -0500
@@ -5,9 +5,9 @@
         <img src="assets/images/dspace-logo.svg" [attr.alt]="'menu.header.image.logo' | translate"/>
       </a>
     </div>
-    <div class="navbar-buttons d-flex flex-grow-1 ml-auto justify-content-end align-items-center gapx-1">
+    <div class="d-flex flex-grow-1 ml-auto justify-content-end align-items-center">
       <ds-themed-search-navbar></ds-themed-search-navbar>
-      <ds-themed-lang-switch></ds-themed-lang-switch>
+      <ds-lang-switch></ds-lang-switch>
       <ds-context-help-toggle></ds-context-help-toggle>
       <ds-themed-auth-nav-menu></ds-themed-auth-nav-menu>
       <ds-impersonate-navbar></ds-impersonate-navbar>
diff -ru dspace/app/header-nav-wrapper/header-navbar-wrapper.component.html unica/app/header-nav-wrapper/header-navbar-wrapper.component.html
--- dspace/app/header-nav-wrapper/header-navbar-wrapper.component.html	2023-12-02 20:20:21.000000000 -0500
+++ unica/app/header-nav-wrapper/header-navbar-wrapper.component.html	2023-11-11 10:23:47.000000000 -0500
@@ -1,3 +1,3 @@
-<div [class.open]="!(isNavBarCollapsed | async)" id="header-navbar-wrapper">
+<div [ngClass]="{'open': !(isNavBarCollapsed | async)}">
   <ds-themed-header></ds-themed-header>
 </div>
diff -ru dspace/app/header-nav-wrapper/header-navbar-wrapper.component.scss unica/app/header-nav-wrapper/header-navbar-wrapper.component.scss
--- dspace/app/header-nav-wrapper/header-navbar-wrapper.component.scss	2023-12-02 20:20:21.000000000 -0500
+++ unica/app/header-nav-wrapper/header-navbar-wrapper.component.scss	2023-11-11 10:23:47.000000000 -0500
@@ -1,7 +0,0 @@
-:host {
-  // The header-navbar-wrapper should not have a z-index, otherwise it would cover the media viewer despite its higher z-index
-  position: relative;
-  div#header-navbar-wrapper {
-    border-bottom: 5px var(--ds-header-navbar-border-bottom-color) solid;
-  }
-}
diff -ru dspace/app/header-nav-wrapper/header-navbar-wrapper.component.ts unica/app/header-nav-wrapper/header-navbar-wrapper.component.ts
--- dspace/app/header-nav-wrapper/header-navbar-wrapper.component.ts	2023-12-02 20:20:21.000000000 -0500
+++ unica/app/header-nav-wrapper/header-navbar-wrapper.component.ts	2023-11-11 10:23:47.000000000 -0500
@@ -6,7 +6,7 @@
  */
 @Component({
   selector: 'ds-header-navbar-wrapper',
-  styleUrls: ['header-navbar-wrapper.component.scss'],
+  styleUrls: ['../../../../app/header-nav-wrapper/header-navbar-wrapper.component.scss'],
   templateUrl: 'header-navbar-wrapper.component.html',
 })
 export class HeaderNavbarWrapperComponent extends BaseComponent {
diff -ru dspace/app/home-page/home-news/home-news.component.html unica/app/home-page/home-news/home-news.component.html
--- dspace/app/home-page/home-news/home-news.component.html	2023-12-02 20:20:21.000000000 -0500
+++ unica/app/home-page/home-news/home-news.component.html	2023-11-11 13:15:45.000000000 -0500
@@ -1,39 +1,15 @@
-<div class="background-image-container mt-ncs">
+<div class="background-image-container">
   <div class="container">
     <div class="jumbotron jumbotron-fluid">
       <div class="d-flex flex-wrap">
         <div>
-          <h1 class="display-3">DSpace 7</h1>
-          <p class="lead">DSpace is the world leading open source repository platform that enables
-            organisations to:</p>
+          <h1 class="display-3">Repositorio Institucional</h1>
+          <p class="lead">Universidad Nacional San Luis Gonzaga</p>          
         </div>
-      </div>
-      <ul>
-        <li>easily ingest documents, audio, video, datasets and their corresponding Dublin Core
-          metadata
-        </li>
-        <li>open up this content to local and global audiences, thanks to the OAI-PMH interface and
-          Google Scholar optimizations
-        </li>
-        <li>issue permanent urls and trustworthy identifiers, including optional integrations with
-          handle.net and DataCite DOI
-        </li>
-      </ul>
-      <p>Join an international community of <a href="https://wiki.lyrasis.org/display/DSPACE/DSpace+Positioning" target="_blank">leading institutions using DSpace</a>.</p>
-      <p>The test user accounts below have their password set to the name of this
-        software in lowercase.</p>
-      <ul>
-        <li>Demo Site Administrator = dspacedemo+admin@gmail.com</li>
-        <li>Demo Community Administrator = dspacedemo+commadmin@gmail.com</li>
-        <li>Demo Collection Administrator = dspacedemo+colladmin@gmail.com</li>
-        <li>Demo Submitter = dspacedemo+submit@gmail.com</li>
-      </ul>
+      </div>      
     </div>
   </div>
   <picture class="background-image">
-    <source type="image/webp" srcset="assets/dspace/images/banner.webp 2000w, assets/dspace/images/banner-half.webp 1200w, assets/dspace/images/banner-tall.webp 768w">
-    <source type="image/jpg" srcset="assets/dspace/images/banner.jpg 2000w, assets/dspace/images/banner-half.jpg 1200w, assets/dspace/images/banner-tall.jpg 768w">
-    <img alt="" [src]="'assets/dspace/images/banner.jpg'"/><!-- without the []="''" Firefox downloads both the fallback and the resolved image -->
+    <img alt="" [src]="'assets/unica/images/banner_unica.jpg'"/><!-- without the []="''" Firefox downloads both the fallback and the resolved image -->
   </picture>
-  <small class="credits">Photo by <a href="https://www.pexels.com/@inspiredimages">@inspiredimages</a></small>
 </div>
diff -ru dspace/app/home-page/home-news/home-news.component.scss unica/app/home-page/home-news/home-news.component.scss
--- dspace/app/home-page/home-news/home-news.component.scss	2023-12-02 20:20:21.000000000 -0500
+++ unica/app/home-page/home-news/home-news.component.scss	2023-11-11 10:23:47.000000000 -0500
@@ -1,5 +1,6 @@
 :host {
   display: block;
+  margin-top: calc(var(--ds-content-spacing) * -1);
 
   div.background-image-container {
     color: white;
diff -ru dspace/app/navbar/navbar.component.html unica/app/navbar/navbar.component.html
--- dspace/app/navbar/navbar.component.html	2023-12-02 20:20:21.000000000 -0500
+++ unica/app/navbar/navbar.component.html	2023-11-11 12:48:21.000000000 -0500
@@ -2,7 +2,7 @@
   class="navbar navbar-expand-md navbar-light p-0 navbar-container" role="navigation" [attr.aria-label]="'nav.main.description' | translate">
   <div class="navbar-inner-container w-100 h-100" [class.container]="!(isXsOrSm$ | async)">
     <a class="navbar-brand my-2" routerLink="/home">
-      <img src="assets/images/dspace-logo.svg" [attr.alt]="'menu.header.image.logo' | translate" />
+      <img src="assets/unica/images/logo_unica.png" [attr.alt]="'menu.header.image.logo' | translate" />
     </a>
 
     <div id="collapsingNav" class="w-100 h-100">
@@ -10,17 +10,15 @@
         <li *ngIf="(isXsOrSm$ | async) && (isAuthenticated$ | async)">
             <ds-user-menu [inExpandableNavbar]="true"></ds-user-menu>
         </li>
-        <li *ngFor="let section of (sections | async)">
+        <ng-container *ngFor="let section of (sections | async)">
           <ng-container *ngComponentOutlet="(sectionMap$ | async).get(section.id)?.component; injector: (sectionMap$ | async).get(section.id)?.injector;"></ng-container>
-        </li>
+        </ng-container>
       </ul>
     </div>
-    <div class="navbar-buttons d-flex align-items-center gapx-1">
-      <ds-themed-search-navbar class="navbar-collapsed"></ds-themed-search-navbar>
-      <ds-themed-lang-switch class="navbar-collapsed"></ds-themed-lang-switch>
-      <ds-context-help-toggle class="navbar-collapsed"></ds-context-help-toggle>
-      <ds-themed-auth-nav-menu class="navbar-collapsed"></ds-themed-auth-nav-menu>
-      <ds-impersonate-navbar class="navbar-collapsed"></ds-impersonate-navbar>
-    </div>
+    <ds-search-navbar class="navbar-collapsed"></ds-search-navbar>
+    <ds-lang-switch class="navbar-collapsed"></ds-lang-switch>
+    <ds-context-help-toggle class="navbar-collapsed"></ds-context-help-toggle>
+    <ds-themed-auth-nav-menu class="navbar-collapsed"></ds-themed-auth-nav-menu>
+    <ds-impersonate-navbar class="navbar-collapsed"></ds-impersonate-navbar>
   </div>
 </nav>
diff -ru dspace/app/navbar/navbar.component.scss unica/app/navbar/navbar.component.scss
--- dspace/app/navbar/navbar.component.scss	2023-12-02 20:20:21.000000000 -0500
+++ unica/app/navbar/navbar.component.scss	2023-11-11 11:28:51.000000000 -0500
@@ -1,27 +1,19 @@
 nav.navbar {
+  border-top: 1px var(--ds-header-navbar-border-top-color) solid;
+  border-bottom: 5px var(--ds-header-navbar-border-bottom-color) solid;
   align-items: baseline;
-
-  .navbar-inner-container {
-    border-top: 1px var(--ds-header-navbar-border-top-color) solid;
-  }
-}
-
-.navbar-nav {
-  background-color: var(--ds-navbar-bg);
 }
 
 /** Mobile menu styling **/
 @media screen and (max-width: map-get($grid-breakpoints, md)-0.02) {
   .navbar {
-    width: 100vw;
+    width: 100%;
     background-color: var(--bs-white);
     position: absolute;
     overflow: hidden;
     height: 0;
-    z-index: var(--ds-nav-z-index);
     &.open {
       height: 100vh; //doesn't matter because wrapper is sticky
-      border-bottom: 5px var(--ds-header-navbar-border-bottom-color) solid; // open navbar covers header-navbar-wrapper border
     }
   }
 }
@@ -57,9 +49,9 @@
 .navbar-nav {
   ::ng-deep a.nav-link {
     color: var(--ds-navbar-link-color);
-
-    &:hover, &:focus {
-      color: var(--ds-navbar-link-color-hover);
-    }
+    font-weight: bold;
+  }
+  ::ng-deep a.nav-link:hover {
+    color: var(--ds-navbar-link-color-hover);
   }
 }
Only in unica/assets/images: alicia.png
Only in unica/assets/images: banner_unica.jpg
Only in unica/assets/images: lareferencia.png
Only in unica/assets/images: logo_unica.png
diff -ru dspace/eager-theme.module.ts unica/eager-theme.module.ts
--- dspace/eager-theme.module.ts	2023-12-02 20:20:21.000000000 -0500
+++ unica/eager-theme.module.ts	2023-11-11 18:27:30.000000000 -0500
@@ -4,12 +4,12 @@
 import { HomeNewsComponent } from './app/home-page/home-news/home-news.component';
 import { NavbarComponent } from './app/navbar/navbar.component';
 import { HeaderComponent } from './app/header/header.component';
+import { FooterComponent } from './app/footer/footer.component';
 import { HeaderNavbarWrapperComponent } from './app/header-nav-wrapper/header-navbar-wrapper.component';
 import { RootModule } from '../../app/root.module';
 import { NavbarModule } from '../../app/navbar/navbar.module';
 import { SharedBrowseByModule } from '../../app/shared/browse-by/shared-browse-by.module';
 import { ResultsBackButtonModule } from '../../app/shared/results-back-button/results-back-button.module';
-
 /**
  * Add components that use a custom decorator to ENTRY_COMPONENTS as well as DECLARATIONS.
  * This will ensure that decorator gets picked up when the app loads
@@ -22,6 +22,7 @@
   HeaderComponent,
   HeaderNavbarWrapperComponent,
   NavbarComponent,
+  FooterComponent,
 ];
 
 @NgModule({
diff -ru dspace/styles/_global-styles.scss unica/styles/_global-styles.scss
--- dspace/styles/_global-styles.scss	2023-12-02 20:20:21.000000000 -0500
+++ unica/styles/_global-styles.scss	2023-11-11 10:23:47.000000000 -0500
@@ -17,7 +17,7 @@
     background-color: var(--bs-primary);
   }
 
-  h4 {
+  h5 {
     font-size: 1.1rem
   }
 }
diff -ru dspace/styles/_theme_css_variable_overrides.scss unica/styles/_theme_css_variable_overrides.scss
--- dspace/styles/_theme_css_variable_overrides.scss	2023-12-02 20:20:21.000000000 -0500
+++ unica/styles/_theme_css_variable_overrides.scss	2023-11-11 20:44:17.000000000 -0500
@@ -6,6 +6,16 @@
   --ds-banner-background-gradient-width: 300px;
   --ds-home-news-link-color: #{$green};
   --ds-home-news-link-hover-color: #{darken($green, 15%)};
-  --ds-header-navbar-border-bottom-color: #{$green};
+  --ds-header-navbar-border-bottom-color: #{$orange};
+  --ds-navbar-link-color: #{$orange};
+  --ds-navbar-link-color-hover: #{$red};
+  --ds-header-icon-color: #{$orange};
+  --ds-breadcrumb-bg: #{$orange} !important;
+  --ds-breadcrumb-link-color: #{$white};
+  --ds-breadcrumb-link-active-color: #{$white};
+  --ds-top-footer-bg: #{$cyan};
+}
+.lead{
+  font-weight: bold !important;
 }
 
diff -ru dspace/styles/_theme_sass_variable_overrides.scss unica/styles/_theme_sass_variable_overrides.scss
--- dspace/styles/_theme_sass_variable_overrides.scss	2023-12-02 20:20:21.000000000 -0500
+++ unica/styles/_theme_sass_variable_overrides.scss	2023-11-11 18:10:29.000000000 -0500
@@ -3,18 +3,18 @@
 // still uses Sass variables internally. So if you want to override bootstrap (or other sass
 // variables) you can do so here. Their CSS counterparts will include the changes you make here
 
-@import url('https://fonts.googleapis.com/css2?family=Nunito:ital,wght@0,200;0,300;0,400;0,600;0,700;0,800;1,200;1,300;1,400;1,600;1,700;1,800&display=swap');
-
-$font-family-sans-serif: 'Nunito', -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, "Helvetica Neue", Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol", "Noto Color Emoji";
+//@import url('https://fonts.googleapis.com/css2?family=OpenSans:ital,wght@0,200;0,300;0,400;0,600;0,700;0,800;1,200;1,300;1,400;1,600;1,700;1,800&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Open+Sans&display=swap');
+$font-family-sans-serif: 'Open Sans', sans-serif;
 
 $navbar-dark-color: #FFFFFF;
 
 /* Reassign color vars to semantic color scheme */
 $blue: #2b4e72 !default;
 $green: #92C642 !default;
-$cyan: #207698 !default;
-$yellow: #ec9433 !default;
-$red: #CF4444 !default;
+$cyan: #004c56 !default;
+$orange: #e58b14 !default;
+$red: #dd0011 !default;
 $dark: #43515f !default;
 
 $gray-800: #343a40 !default;
@@ -30,11 +30,11 @@
 $yiq-contrasted-threshold:  170 !default;
 
 $theme-colors: (
-        primary: $dark,
+        primary: $cyan,
         secondary: $gray-700,
         success: $green,
         info: $cyan,
-        warning: $yellow,
+        warning: $orange,
         danger: $red,
         light: $gray-100,
         dark: $dark
